microcosm_flask/conventions/relation.py

```diff
@@ -41,11 +42,56 @@ def configure_createfor(self, ns, definition):
         @response(definition.response_schema)
         def create(**path_data):
             request_data = load_request_data(definition.request_schema)
-            response_data = definition.func(**merge_data(path_data, request_data))
+            response_data = require_response_data(definition.func(**merge_data(path_data, request_data)))
             return dump_response_data(definition.response_schema, response_data, Operation.CreateFor.value.default_code)
 
         create.__doc__ = "Create a new {} relative to a {}".format(pluralize(ns.object_name), ns.subject_name)
 
+    def configure_deletefor(self, ns, definition):
+        """
+        Register a delete-for relation endpoint.
+
+        The definition's func should be a delete function, which must:
+        - accept kwargs for path data
+        - return truthy/falsey
+
+        :param ns: the namespace
+        :param definition: the endpoint definition
+
+        """
+        @self.graph.route(ns.instance_path, Operation.DeleteFor, ns)
+        def delete(**path_data):
+            require_response_data(definition.func(**path_data))
+            return "", Operation.DeleteFor.value.default_code
+
+        delete.__doc__ = "Delete a {} relative to a {}".format(pluralize(ns.object_name), ns.subject_name)
+
+    def configure_replacefor(self, ns, definition):
```

Comment:
I'd like to be convinced that this endpoint makes sense as a convention.

In my mind, we only use replace when we already have an object id (whether or not it exists in the database yet). I don't see why we'd operate on the relation in this case instead of the regular CRUD location.